ui/helpers/job.js

```diff
@@ -157,12 +154,10 @@ export const findJobInstance = function findJobInstance(jobId, scrollTo) {
   const jobEl = document.querySelector(
     `#push-list button[data-job-id='${jobId}']`,
   );
-  if (jobEl) {
-    if (scrollTo) {
-      scrollToElement(jobEl);
-    }
-    return findInstance(jobEl);
+  if (scrollTo) {
+    scrollToElement(jobEl);
```

Comment:
Can the `scrollToElement(jobEl)` handle undefined elements? (Since you're removing `if (jobEl) {`)

Code in question:

```javascript
// Scroll the element into view.
export const scrollToElement = function scrollToElement(el) {
  if (!isOnScreen(el)) {
    el.scrollIntoView({ behavior: 'smooth', block: 'center' });
  }
};

// Check if the element is visible on screen or not.
const isOnScreen = function isOnScreen(el) {
  const bounding = el.getBoundingClientRect();
  const offset = el.getBoundingClientRect();
  const top = offset.top + document.body.scrollTop;
  const bottom = top + el.offsetHeight;
  return top >= bounding.bottom && bottom <= bounding.top;
};
```

Reply:
I will look into this!